retro/init_obj.py

```diff
@@ -597,18 +706,18 @@ def iterate_file(fpath, start=None, stop=None, step=None):
             # Note that memory mapping the file is useful for not consuming too
             # much memory, and also might be essential in the future if we
             # write recos directly to the {reco}.npy file
-            events = np.load(fpath, mmap_mode='r')
+            events = np.load(fpath) #, mmap_mode='r')
```

Comment:
IIRC I got a "too many open files" error, so I disabled memory mapping this file to test and forgot to remove / change the above comment (I haven't seen that error since; not 100% sure this is what fixed it, but `events.npy` is not a big file so reading all at once probably isn't a big deal).

I'll modify to remove the misleading comment above and remove the commented-out bit in a couple of minutes.

Reply:
made a future-proof modification here (function defaults to not use memory mapping, but can easily do so we want): https://github.com/philippeller/retro/pull/75/commits/0d8d3cba431d83c5a42fbb145eefbb090bcfcb25